src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -472,9 +472,28 @@ def delete_lock(name, resource_group_name=None, resource_provider_namespace=None
 
 def create_lock(name, resource_group_name=None, resource_provider_namespace=None,
                 parent_resource_path=None, resource_type=None, resource_name=None,
-                level=None, notes=None, lock_id=None, lock_type=None):
+                level=None, notes=None, lock_id=None):
```

Comment:
This is only referenced in source, user will never see this. SDK calls it 'level' this is the place where the user name and the SDK name are linked up.